LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright 2018 Northern.tech AS
+Copyright 2019 Northern.tech AS
```

Comment:
Something is odd here. We should already have these changes in master, see PR #26

Reply:
You're right, I had not rebased. Fixed.